### PR TITLE
chore: add win10 pipelinerun example

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
chore: add win10 efi pipelinerun example

**Release note**:
```
chore: add win10 efi pipelinerun example
```

### CODE REVIEW

File: release/pipelines/windows-efi-installer/README.md [hunk 1]
```diff
@@ -110,6 +110,47 @@ EOF
...
+metadata:
+    generateName: windows10-installer-run-
+spec:
+    params:
+    -   name: winImageDownloadURL
+        value: ${WIN_IMAGE_DOWNLOAD_URL}
+    -   name: preferenceName
+        value: windows.10
+    -   name: autounattendConfigMapName
+        value: windows10-efi-autounattend
```
Comment (0xFelix) [inline release/pipelines/windows-efi-installer/README.md:122]: Please remind me, does the config map contain virtio drives? If so you could use `windows.10.virtio` above.

File: release/pipelines/windows-efi-installer/README.md
Comment (0xFelix) [inline release/pipelines/windows-efi-installer/README.md:1]: Indentation in this file looks off.

File: release/pipelines/windows-efi-installer/pipelineruns/pipelineruns.yaml [hunk 2]
```diff
@@ -30,6 +30,44 @@ spec:
...
 apiVersion: tekton.dev/v1
 kind: PipelineRun
+metadata:
+  generateName: windows10-installer-run-
+spec:
+  params:
+    - name: winImageDownloadURL
+      value: ${WIN_IMAGE_DOWNLOAD_URL}
+    - name: preferenceName
+      value: windows.10
```
Comment (0xFelix) [inline release/pipelines/windows-efi-installer/pipelineruns/pipelineruns.yaml:40]: Same

File: templates-pipelines/windows-efi-installer/pipelineruns/pipelineruns.yaml [hunk 3]
```diff
@@ -30,6 +30,44 @@ spec:
...
 apiVersion: tekton.dev/v1
 kind: PipelineRun
+metadata:
+  generateName: windows10-installer-run-
+spec:
+  params:
+    - name: winImageDownloadURL
+      value: ${WIN_IMAGE_DOWNLOAD_URL}
+    - name: preferenceName
+      value: windows.10
```
Comment (0xFelix) [inline templates-pipelines/windows-efi-installer/pipelineruns/pipelineruns.yaml:40]: Same

File: release/pipelines/windows-efi-installer/README.md (hunk 1 above)
Comment (ksimon1) [reply release/pipelines/windows-efi-installer/README.md:122]: yes, I will fix it

File: release/pipelines/windows-efi-installer/README.md
Comment (ksimon1) [reply release/pipelines/windows-efi-installer/README.md:1]: Where exactly?

File: release/pipelines/windows-efi-installer/README.md [hunk 4]
```diff
@@ -110,6 +110,47 @@ EOF
 oc create -f - <<EOF
 apiVersion: tekton.dev/v1
 kind: PipelineRun
+metadata:
+    generateName: windows10-installer-run-
+spec:
+    params:
+    -   name: winImageDownloadURL
```
Comment (0xFelix) [inline release/pipelines/windows-efi-installer/README.md:117]: e.g. here between the `-` and `name`, also it is using 4 spaces for indendation.

Comment (ksimon1) [reply release/pipelines/windows-efi-installer/README.md:117]: This is done by ansible

**Comment by ksimon1** [comment]
The change which will modify preference to virtio for other OSes will be done in follow up PR

File: release/pipelines/windows-efi-installer/README.md (hunk 4 above)
Comment (0xFelix) [reply release/pipelines/windows-efi-installer/README.md:117]: Weird but OK...

**Review by 0xFelix — APPROVED** [review]
/approve

**Pull request merged**